src/data/queries/tests/storyListing.test.tsx

```diff
@@ -141,9 +141,8 @@ describe('data function for StoryListing', () => {
           field_media: null,
           field_related_links: {
             id: '1',
-            field_link: null,
-            field_links: [],
-            field_section_header: null,
+            field_va_paragraphs: [],
+            field_title: null,
```

Comment:
The `field_office` uses the `node--health_care_region_page`, which maps to the `NodeHealthCareRegionPage` static type. This static type was using the wrong static type for its `field_related_links`, so this mock data was the wrong shape.

This is what we get for not pulling mock data from an authoritative source. 🤷